m&m (import me).xml

```diff
@@ -55554,8 +55554,8 @@ end
 
 -- downloads the system &amp; updates the system version saved
 function mm.downloadnewsystem(newversion)
-  mm.downloadedsystem = downloadfolder..string.format(&quot;%s m&amp;m.zip&quot;, mm.me.name)
-  local downloadurl = [[http://vadisystems.com/mm-r/r/]]..mm.me.name..[[ m&amp;m.zip]]
+  mm.downloadedsystem = downloadfolder..string.format(&quot;%s m&amp;m.zip&quot;, mm.me.class)
+  local downloadurl = [[http://m-mf.github.io/m-mf/r/]]..mm.me.class..[[ m&amp;m.zip]]
```

Comment:
URL should be [[http://m-mf.github.io/m-mf/release/]]..mm.me.class..[[ m&amp;m.zip]] here